src/game/playerShooting.js

```diff
@@ -0,0 +1,39 @@
+// import { computerState, computerShips, computerEmptyFields } from "./game";
+
+function hitField() {
+  event.target.removeEventListener("click", hitField);
+  event.target.style.cursor = "not-allowed";
+  const fieldName = event.target.dataset.fieldName;
+  let fieldindex = computerShips.findIndex(
+    (ship) => ship.fieldName === fieldName
+  );
+  if (fieldindex >= 0) {
+    const shipName = computerShips[fieldindex].shipName;
+    computerShips.splice(fieldindex, 1);
+    let ship = computerState.ships.find((ship) => ship.shipName === shipName);
+    let field = ship.fields.findIndex((field) => field.name === fieldName);
+    ship.fields[field].hit = true;
+    const isSunk = ship.fields.every((field) => field.hit === true);
+    ship.sunk = isSunk;
+    const lostAllShieps = computerState.ships.every(
+      (ship) => ship.sunk === true
+    );
+    computerState.lostAllShips = lostAllShieps;
+  } else {
+    fieldindex = computerEmptyFields.findIndex(
+      (field) => field.fieldName === fieldName
+    );
+    if (fieldindex >= 0) {
+      computerEmptyFields.splice(fieldindex, 1);
+    } else return;
+  }
+}
+
+export function addShooting() {
```

Comment:
`addShooting` sounds a bit weird - what does this method do? It adds a field possibility to be targeted or what?

Reply:
It means that i add possibility for field to be targeted. Maybe `addPossibilityToTargetField` ? In my opinion it's a bit long. Have you got any ideas to call this function?

Reply:
`makeFieldTargetable`